webapp/src/components/kanban/kanban.tsx

```diff
@@ -91,7 +91,11 @@ const Kanban = (props: Props) => {
         }
 
         await mutator.insertPropertyOption(board.id, board.cardProperties, groupByProperty!, option, 'add group')
-    }, [board, groupByProperty])
+
+        // Ugly hack to add the new column to the left
+        // Simulate moving it to the place of current first column
+        await onDropToColumn(option, undefined, visibleGroups[0].option, true)
```

Comment:
if that works slow, can we call some downstream function used in `onDropToColumn` instead?
`onDropToColumn` wraps the move in `mutator.performAsUndoGroup` which we don't need here, so maybe we could call `mutator.changeViewCardOrder` instead?

Reply:
Here will be used this part of `onDropToColumn`:
`else if (dstOption) {` which wraps `mutator.changeViewVisibleOptionIds`, and before does preparations which doesn't look unnecessary to me.
Shortcut may be found in `dragAndDropRearrange` function, cause we know what and where we are moving, but I'm note sure if it would make the process significantly faster

Reply:
I'l take a look and try to create a separate function for this specific column move, It shouldn't be difficult. But during the week :)